app/controllers/LanguageController.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.AppConfig
+import javax.inject.Inject
+import play.api.i18n.{I18nSupport, Lang, MessagesApi}
+import play.api.mvc.{Action, AnyContent, Call}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.play.language.LanguageUtils
+
+
+class LanguageController @Inject()(val appConfig: AppConfig,
+                                   val messagesApi: MessagesApi) extends FrontendController with I18nSupport {
+
+  def langToCall: String => Call = appConfig.routeToSwitchLanguage
+
+  protected[controllers] def fallbackURL: String = controllers.routes.HelloWorldController.helloWorld().url
```

Comment:
This may need to be changed as the hello world controller is being removed before we go live I belive , it may need to redirect the user to a page that is not in this project

Reply:
Changed the redirect url to ChooseDatesController